public void testDBException() throws Exception {
    this.factory = EvilDAOFactory.getEvilInstance();
    factor = new PriorDiagnosisFactor(factory, 2L, 250.3, 487);
    assertFalse(factor.hasFactor());
}